frontend/src/views/ensemble/barline/Barline.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box } from "@mui/material";
+import _ from "lodash";
+import SingleNote from "./SingleNote";
+import { UserID } from "common/dist";
+
+// consts for calculating barline, not including volume bar
+const BARLINE_WIDTH = 32; // 8 bars of 4 quarter notes = 32 note placements
+const BARLINE_HEIGHT = 8; // 8 Notes from C -> C
```

Comment:
You can probably replace these with the Barline static variables. Just make sure to import Barline. 

`Barline.ROWS`
`Barline.COLS`

This is to make sure we are not duplicating sources of truth. You can move these comments to the Barline class in common.